pcweb/pages/docs/component_lib/forms/input.py

```diff
@@ -126,15 +123,20 @@ def render_input():
             context=True,
         ),
         doctext(
-            "Alternatively, the ",
-            rx.code("rx.debounce_input"),
-            " component can wrap an ",
-            rx.code("rx.input"),
-            " to avoid sending state updates ",
-            "to the backend while the user is still typing. ",
+            "Behind the scene, the input component is implemented ",
+            "using debounced input to avoid sending individual state ",
+            "updates per character to the backend while the user is ",
+            "still typing. ",
             "This allows a state var to directly control the ",
             rx.code("value"),
             " prop from the backend without the user experiencing input lag. ",
+            "For advanced use cases, you can tune the debounce delay ",
+            "by setting the ",
+            rx.code("debounce_timeout"),
+            " prop. ",
+            "You can find examples of how it is used in ",
+            doclink("DebouncedInput", href="/docs/library/forms/debounceinput"),
```

Comment:
```suggestion
            doclink("DebounceInput", href=render_debounceinput.path),
```

Reply:
This is what I tried first, but this doesn't seem to work. There's those returning a page or something, those have the path attribute

Reply:
ah, it looks like `render_debounceinput` might be missing the `@docpage` decorator? not sure exactly.

Reply:
adding the decorator caused some other errors for me, didn't look different from other existing examples with the decorator, strange.